x-pack/plugins/security_solution/server/lib/telemetry/__mocks__/index.ts

```diff
@@ -85,6 +85,7 @@ export const createMockTelemetryReceiver = (
     openPointInTime: jest.fn().mockReturnValue(Promise.resolve('test-pit-id')),
     getAlertsIndex: jest.fn().mockReturnValue('alerts-*'),
     fetchDiagnosticAlertsBatch: jest.fn().mockReturnValue(diagnosticsAlert ?? jest.fn()),
+    getExperimentalFeatures: jest.fn().mockReturnValue(undefined),
```

Comment:
Should we set the default value to `false or undefined` and avoid adding these changes here in `telemetry` code? Or do we need this here too?

Reply:
undefined is the type that comes from experimentalFeatures in the first place, so I wasn't planning to default that to false. Where would you want me to default to `undefined` ?  

Regarding telemetry... yeah I would prefer not to make changes in there, but it was required because the function using schema was used in there too...

Reply:
If we need that change in telemetry (I think we need it) then I'm ok with the current approach. Thanks!